### DISCUSSION TITLE
Access Denied from Imperva check

### DISCUSSION THREAD

**Discussion by fw2155** [ultrafunkamsterdam/undetected-chromedriver#1809] [Q&A] [answered]
In my if __name__ == "__main__":
```
s = Service(ChromeDriverManager().install())
    s.start()
    browser = uc.Chrome(service=s, version_main=123)
    url = "https://redeem.tcg.pokemon.com/en-us/"
    browser.get(url)
```
    
   I am still have access denied when browsing to that website though using uc. Any thought on how to solve?
   Thank you!

**Comment by KebabLord** [comment, marked as answer]
i can access it with undetected chromedriver

make sure you are using the latest version;
``` py -m pip install -U undetected-chromedriver ```

also you don't have to install webdriver everytime.
```py
import undetected_chromedriver as uc
driver = uc.Chrome()
url = "https://redeem.tcg.pokemon.com/en-us/"
driver.get(url)
input("PRESS ENTER TO CLOSE THE SCRIPT")
```
